controller/src/test/java/org/jboss/as/controller/persistence/PersistanceResourceTestCase.java

```diff
@@ -353,6 +353,12 @@ public void testPersistentNameFromOutsideConfigDirectory() throws Exception {
         ConfigurationFile configurationFile = new ConfigurationFile(standardDir, "standard.xml", file.getAbsolutePath(), true);
     }
 
+    @Test(expected = IllegalStateException.class)
+    public void testRelativePathFromOutsideConfigDirectory() throws Exception {
+        File file = createFile(externalDir, "standard.xml", "test");
+        ConfigurationFile configurationFile = new ConfigurationFile(standardDir, "standard.xml", "../external/standard.xml", true);
+    }
```

Comment:
You can easily add a test to verify that there is no exception for InteractionPolicy.READ_ONLY, that will help us to somehow document this expected behavior.